.........C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py:293: FutureWarning: Using a non-tuple sequence for multidimensional indexing is deprecated; use `arr[tuple(seq)]` instead of `arr[seq]`. In the future this will be interpreted as an array index, `arr[np.array(seq)]`, which will result either in an error or a different result.
  zi = np.argpartition(x, -k, axis=axis)[idx]
....................................................................................................................................................................................................................................................................................................................................................................................................................C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py:285: FutureWarning: Using a non-tuple sequence for multidimensional indexing is deprecated; use `arr[tuple(seq)]` instead of `arr[seq]`. In the future this will be interpreted as an array index, `arr[np.array(seq)]`, which will result either in an error or a different result.
  zi = np.argpartition(x, -k, axis=axis)[idx]
...................................................................E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..E..C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py:282: FutureWarning: Using a non-tuple sequence for multidimensional indexing is deprecated; use `arr[tuple(seq)]` instead of `arr[seq]`. In the future this will be interpreted as an array index, `arr[np.array(seq)]`, which will result either in an error or a different result.
  zv = np.partition(x, -k, axis=axis)[idx]
C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py:285: FutureWarning: Using a non-tuple sequence for multidimensional indexing is deprecated; use `arr[tuple(seq)]` instead of `arr[seq]`. In the future this will be interpreted as an array index, `arr[np.array(seq)]`, which will result either in an error or a different result.
  zi = np.argpartition(x, -k, axis=axis)[idx]
......C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py:282: FutureWarning: Using a non-tuple sequence for multidimensional indexing is deprecated; use `arr[tuple(seq)]` instead of `arr[seq]`. In the future this will be interpreted as an array index, `arr[np.array(seq)]`, which will result either in an error or a different result.
  zv = np.partition(x, -k, axis=axis)[idx]
C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py:285: FutureWarning: Using a non-tuple sequence for multidimensional indexing is deprecated; use `arr[tuple(seq)]` instead of `arr[seq]`. In the future this will be interpreted as an array index, `arr[np.array(seq)]`, which will result either in an error or a different result.
  zi = np.argpartition(x, -k, axis=axis)[idx]
......C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py:282: FutureWarning: Using a non-tuple sequence for multidimensional indexing is deprecated; use `arr[tuple(seq)]` instead of `arr[seq]`. In the future this will be interpreted as an array index, `arr[np.array(seq)]`, which will result either in an error or a different result.
  zv = np.partition(x, -k, axis=axis)[idx]
C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py:285: FutureWarning: Using a non-tuple sequence for multidimensional indexing is deprecated; use `arr[tuple(seq)]` instead of `arr[seq]`. In the future this will be interpreted as an array index, `arr[np.array(seq)]`, which will result either in an error or a different result.
  zi = np.argpartition(x, -k, axis=axis)[idx]
......C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py:282: FutureWarning: Using a non-tuple sequence for multidimensional indexing is deprecated; use `arr[tuple(seq)]` instead of `arr[seq]`. In the future this will be interpreted as an array index, `arr[np.array(seq)]`, which will result either in an error or a different result.
  zv = np.partition(x, -k, axis=axis)[idx]
C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py:285: FutureWarning: Using a non-tuple sequence for multidimensional indexing is deprecated; use `arr[tuple(seq)]` instead of `arr[seq]`. In the future this will be interpreted as an array index, `arr[np.array(seq)]`, which will result either in an error or a different result.
  zi = np.argpartition(x, -k, axis=axis)[idx]
.........................................................................................................................................C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py:285: FutureWarning: Using a non-tuple sequence for multidimensional indexing is deprecated; use `arr[tuple(seq)]` instead of `arr[seq]`. In the future this will be interpreted as an array index, `arr[np.array(seq)]`, which will result either in an error or a different result.
  zi = np.argpartition(x, -k, axis=axis)[idx]
...................
======================================================================
ERROR: test_combined_sanity_002_int8 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_005_int8 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_008_int8 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_011_int8 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_014_int8 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_017_int8 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_020_int8 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_023_int8 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_026_int16 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_029_int16 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_032_int16 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_035_int16 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_038_int16 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_041_int16 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_044_int16 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_047_int16 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_050_int32 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_053_int32 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_056_int32 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_059_int32 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_062_int32 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_065_int32 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_068_int32 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_071_int32 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_074_int64 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_077_int64 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_080_int64 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_083_int64 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_086_int64 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_089_int64 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_092_int64 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_095_int64 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_098_uint8 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_101_uint8 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_104_uint8 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_107_uint8 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_110_uint8 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_113_uint8 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_116_uint8 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_119_uint8 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_122_uint16 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_125_uint16 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_128_uint16 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_131_uint16 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_134_uint16 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_137_uint16 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_140_uint16 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_143_uint16 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_146_uint32 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_149_uint32 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_152_uint32 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_155_uint32 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_158_uint32 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_161_uint32 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_164_uint32 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_167_uint32 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_170_uint64 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_173_uint64 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_176_uint64 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_179_uint64 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_182_uint64 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_185_uint64 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_188_uint64 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_191_uint64 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_194_float16 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_197_float16 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_200_float16 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_203_float16 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_206_float16 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_209_float16 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_212_float16 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_215_float16 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_218_float32 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_221_float32 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_224_float32 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_227_float32 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_230_float32 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_233_float32 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_236_float32 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_239_float32 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_242_float64 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_245_float64 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_248_float64 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_251_float64 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_254_float64 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_257_float64 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_260_float64 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

======================================================================
ERROR: test_combined_sanity_263_float64 (test_sort.Test_TopK)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\parameterized\parameterized.py", line 530, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "C:\Users\mutation\Desktop\theano-testcase\tests\test_sort.py", line 269, in test_combined_sanity
    yv, yi = topk_and_argtopk(x, 1, axis=axis, sorted=sorted, idx_dtype=idx_dtype)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 563, in topk_and_argtopk
    idx_dtype=idx_dtype)(x, kth)
  File "C:\ProgramData\Anaconda3\envs\keras\lib\site-packages\theano\tensor\sort.py", line 367, in __init__
    '"axis" parameter must be integer, got "%s"' % type(axis))
TypeError: "axis" parameter must be integer, got "<class 'NoneType'>"

----------------------------------------------------------------------
Ran 918 tests in 56.299s

FAILED (errors=88)
